backend/internal/service/handler/spotistics/clientcreds.go

```diff
@@ -0,0 +1,30 @@
+package spotistics
+
+import (
+	"context"
+	"log"
+	"os"
+
+	spotifyauth "github.com/zmb3/spotify/v2/auth"
+
+	"github.com/zmb3/spotify/v2"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+func clientcreds() spotify.Client {
```

Comment:
you probably want to store the credentials in the struct then have this as a private method. that way you don't have to construct the config every time. also, i would probably create middleware along the lines of `WithSpotify` to handle this, store the client in context, the continue.